lib/engine/step/waterfall_auction.rb

```diff
@@ -121,6 +121,7 @@ def resolve_bids_for_company(company)
           @log << "#{@auctioning.name} goes up for auction" if is_new_auction
         end
 
+        @game.next_turn!
```

Comment:
I'm not sure this catches all the edge cases. For example, I don't think it catches a player who passes when there is nothing actively being auctioned. Would adding `@game.next_turn!` to `process_pass` and `process_bid` bid more robust?